src/server/controller/RegistrationController.js

```diff
@@ -52,14 +52,17 @@ const sendErr = (res, errMsg, err) => {
  * @param res
  * @param next
  */
-// TODO: Improve me, use check on whether there is any active event
-registration.lock = (req, res, next) => {
-  if (Date.now() > new Date('Tue May 09 2018 05:00:00 GMT+0200 (CEST)')) {
-    res.redirect('/closed');
-  } else {
+
+registration.lock = (req, res, next) => co(function*() {
+  const events = yield api.getAllEvents();
+  if (events.find(event => event.openForRegistration)) {
     next();
+  } else {
+    res.redirect('/closed');
   }
-};
+}).catch(err => {
+  sendErr(res, err.message, err);
+});
```

Comment:
Don't use a co function here but a generator function `registration.lock = function*() { ... }` The router will wrap this automatically